src/main/java/uk/gov/hmcts/divorce/document/DocAssemblyService.java

```diff
@@ -37,11 +38,12 @@
     public DocumentInfo renderDocument(
         CaseData caseData,
         Long caseId,
+        LocalDateTime createDate,
```

Comment:
this is date time and not date. Do we know how this is going to be shown on the draft petition document ?
I think it should be just date

Reply:
I pushed a commit that makes it clear that LocalDate is used